docs/note.md

```
@@ -0,0 +1,104 @@
+Let's combine the two components @ContentErrorPane.vue and @CliBar.vue together, and implement an amazing functionality step by step here is the thing:
+- when I click btn debugWithAI, the @CliBar.vue will popup from bottom to the half height of the tab pane, the data-table on it's top will shrink to half height
+- the session name is composed of `resource kind: resource name:session.address : session.port : session.cmds`
+- The clicked item's info will sent as init prompt to the chat box
+- After the response msg is received, there are 2 options(Copy and Apply) can be shown up at the top right of the msg box when the mouse is hoved on any place on the response msg box, I can copy it and paste to the right terminal to be executed. And I can also click Apply btn to execute the command in the response box in the right terminal
+- The popuped CliBar is draggable, and can be closed by clicking the close button on the top right of the CliBar, and when it's displayed, the data-table in @ContentErrorPane.vue is also visible(include the pagination on the bottom right), I can opreate it normally
+
+Let's continue to implement @CliBar.vue 's functionality:
+- I want the content of the chat box and the terminal can be keeped, when I swith back from others tab to current tab, so that I can continue to operate in the chat and terminal with previous context
+
+
+### CliBar:
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)